target/linux/generic/hack-6.12/251-kconfig.patch

```diff
@@ -146,7 +146,16 @@ Signed-off-by: John Crispin <john@phrozen.org>
  	bool
 --- a/net/Kconfig
 +++ b/net/Kconfig
-@@ -467,7 +467,7 @@ config NET_DEVLINK
+@@ -68,7 +68,7 @@ config SKB_EXTENSIONS
+ 
+ config NET_DEVMEM
+ 	def_bool y
+-	depends on DMA_SHARED_BUFFER
++	select DMA_SHARED_BUFFER
```

Comment:
This fix is not correct. Please cherry-pick and squash this patch:
https://github.com/DragonBluep/openwrt/commit/1c35886179c7c94741381caeb0caac4d75261c56